libr/core/cmd_project.c

```diff
@@ -158,6 +159,27 @@ static int cmd_project(void *data, const char *input) {
 				eprintf ("Usage: `Pnj` or `Pnj ...`\n");
 			}
 			break;
+		case 'x':
+			char *str = r_core_project_notes_file (core, fileproject);
```

Comment:
dont put vars in the middle of case. it will not compile with old compilers (by old i mean 1 month old). put that logic in a separate function